game.js

```diff
@@ -0,0 +1,80 @@
+let playerPoints = 0;
+let computerPoints = 0;
+let choices = ["Rock", "Paper", "Scissors"];
+
+// Here we taking the choice of computer
+getComputerChoice = () => choices[Math.floor(Math.random()*choices.length)]
```

Comment:
It is not a good practice to have variables or functions without specifying the declarations (`var`/`let`/`const`) explicitly.

Since functions definitions are const - don't change in the entire scope, functions are typed as `const`

Reply:
Okay.Done